modules/webp-uploads/load.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * WebP-Uploads Module
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ *
+ * Module Name: WebP uploads.
+ * Description: Uses WebP as the default format for new JPEG image uploads if the server supports it.
+ * Focus: images
+ * Experimental: No
+ */
+
+// Only enable if the server supports WebP.
+if ( wp_image_editor_supports( array( 'mime_type' => 'image/webp' ) ) ) {
```

Comment:
Is this check cheap to do? Wondering since it's run unconditionally here, which would mean the check is made on every page load, even in the frontend.

I think it may be more efficient to add the filter unconditionally, and then do the check within the filter callback. This way the check is only made when it's actually needed.

Reply:
👍🏼